tox.ini

```diff
@@ -56,13 +60,13 @@ commands =
     pytest --cov buku -vv -m "not non_tox" {posargs}
 
 [testenv:pylint]
-basepython = py311
+basepython = {env:BASEPYTHON:py311}
```

Comment:
Windows doesn't seem to support Python detection correctly (likely due to not having a fixed file structure).

This allows to override configured Python version with an env variable.